utils/eslint-runner.mjs

```diff
@@ -0,0 +1,35 @@
+import fs from 'fs';
```

Comment:
I suggest using typescript here (actually for all scripts).

We have to add tsconfig here (smth like `./utils/tsconfig.json`) and use 'ts-node' to call scripts.
It's simple and allows us to take advantage of type-checking which is good for maintainability 


```suggestion
        "lint-demos": "ts-node utils/eslint-runner",
```